### PR TITLE
DROOLS-6010 : Not able to use a space within quotes in guided DecisionTable.

### PR DESCRIPTION
**Pull request by Rikkola**
**JIRA**:  

[DROOLS-6010](https://issues.redhat.com/browse/DROOLS-6010)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by Rikkola** [comment]
`org.drools.compiler.integrationtests.AccumulateTest.testDoubleAccumulate[KieBase type=CLOUD_IDENTITY_MODEL_FLOW]
org.drools.compiler.integrationtests.AccumulateTest.testDoubleAccumulate[KieBase type=CLOUD_IDENTITY_MODEL_PATTERN]`

Jenkins retest this

**Comment by Rikkola** [comment]
Jenkins retest this

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java [hunk 1]
```diff
@@ -2859,15 +2859,15 @@ public void testLHSInOperator() {
...
+                new Object[]{1l, "", "desc", "Pupa, Brains, \"John, Snow\", \" \"", "55, 66"},
                 new Object[]{2l, "", "desc", "", ""}
         }));
 
         GuidedDTDRLPersistence p = GuidedDTDRLPersistence.getInstance();
         String drl = p.marshal(dt);
 
         int index = -1;
-        index = drl.indexOf("Smurf( name in ( \"Pupa\", \"Brains\", \"John, Snow\" ) , age in ( 55, 66 ) )");
+        index = drl.indexOf("Smurf( name in ( \"Pupa\", \"Brains\", \"John, Snow\", \" \" ) , age in ( 55, 66 ) )");
```
Comment (jomarko) [inline drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java]: Thank you for a perfect reproducer @Rikkola , could we please increase the coverage by https://github.com/Rikkola/drools/pull/10 ?

**Review by jomarko — COMMENTED** [review]
One comment to code, double checking manually.

**Comment by jomarko** [comment]
Tried manually and works correctly. Once coverage increase is merged, will approve.

File: drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java (hunk 1 above)
Comment (Rikkola) [reply drools-workbench-models/drools-workbench-models-guided-dtable/src/test/java/org/drools/workbench/models/guided/dtable/backend/GuidedDTDRLPersistenceTest.java]: @jomarko Thank you for the PR, merged it in.

**Review by jomarko — APPROVED** [review]

**Comment by jomarko** [comment]
@Rikkola could you start also full downstream build please? So we are sure tests in drools-wb are fine.

**Review by karreiro — APPROVED** [review]
Thank you, @Rikkola LGTM :-)

**Comment by Rikkola** [comment]
Jenkins run fdb

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Can I get a merge please.

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Can I get a merge please. FDB passed, so no need for CDB to turn green.

**Pull request merged**